src/it/exclusions/pom.xml

```diff
@@ -68,6 +74,9 @@ SOFTWARE.
                 <exclusion>
                   JTCOP.RuleAssertionMessage
                 </exclusion>
+                <exclusion>
+                  JTCOP.RuleNotContainsTestWord
```

Comment:
@h1alexbel Can we just rename the test method `testsSomething` -> `checksSomething`? Then we won't need this `exclusion`.